winui/AI-AssistView/ResponseToolBar.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: post
+title: Response Toolbar in WinUI AI AssistView control | Syncfusion
+description: Learn about Response Toolbar with the Syncfusion WinUI AI AssistView (SfAIAssistView) control with its basic features.
+platform: WinUI
+control: AI AssistView
+documentation: ug
+---
+
+# Response Toolbar in WinUI AI AssistView 
+
+The [SfAIAssistView](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Chat.SfAIAssistView.html) control includes a **Response Toolbar** feature that enables users to customize the response toolbar in chat interactions. By default, the toolbar offers built-in options such as Copy, Regenerate, Like, and Dislike. Additionally, you can add custom toolbar items as needed.
+
+## ResponseToolbarItem
+
+The **ResponseToolbarItem** class is used to define and add items in the response toolbar of chat responses for SfAIAssistView. The following properties are available in this class:
+
+- **Index**: Specifies the internal order or position of the toolbar item
+- **IsEnabled**: Specifies whether the toolbar item is enabled (interactive) or disabled (non-interactive).
+- **Tooltip**: Sets the tooltip text displayed when the user hovers over the toolbar item.
+- **Visible**: Indicates whether the toolbar item is visible.
+- **ItemType**: Specifies the type of the toolbar item. This is used to distinguish between built-in and custom items.
+- **ItemTemplate**: Allows you to set a custom template for rendering the toolbar item.
+
+{% tabs %}
+{% highlight xaml %}
+
+ <Grid x:Name="grid1">
+    <syncfusion:SfAIAssistView x:Name="aiAssistView" CurrentUser="{Binding CurrentUser}"  
+                               Messages="{Binding Chats}" >
+    </syncfusion:SfAIAssistView>
+</Grid>
+
+{% endhighlight %} 
+{% endtabs %}
+
+![ResponseToolBar feature in WinUI SfAIAssistView control](aiassistview_images/winui_aiassistview_responsetoolbar.png)
+
+## Response Toolbar Visibility
+
+The **IsResponseToolbarVisible** property of the [SfAIAssistView](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Chat.SfAIAssistView.html) control determines whether the response toolbar is displayed below each chat message. When set to false, the response toolbar is hidden. By default, this property is set to true.
```

Comment:
@Aravindhanperiyasamy19 

The IsResponseToolbarVisible property of the SfAIAssistView control determines whether the response toolbar is displayed below each chat items responses. When set to false, the response toolbar is hidden. By default, this property is set to true.